ext/js/data/options-util.js

```diff
@@ -1173,6 +1174,18 @@ export class OptionsUtil {
         return options;
     }
 
+    /**
+     * @type {import('options-util').ModernUpdateFunctionAsync}
```

Comment:
Sync not Async

Reply:
Although I guess my next PR will entirely do away with this distinction so I suppose it doesn't matter.